run-go-fmt.sh

```diff
@@ -4,6 +4,6 @@
 #
 set -e
 
-exec 5>&1
-output="$(gofmt -l -w "$@" | tee /dev/fd/5)"
-[[ -z "$output" ]]
+output="$(gofmt -l -w "$@")"
+[[ -z "$output" ]] && exit 0
+echo "files formatted, please re-add to commit"
```

Comment:
I think the one difference with this approach is that previous the list of modified files would be printed, and now it is now. If all the files were added with `git add` then it may be obvious which files were changed, but if the commit is run using `git commit -a` it may not be obvious.

I think we could restore printing of the filenames by adding an `echo "$output"` to both of these scripts.